ServiceMonitoringFunction/tst/com.amazon.vas.serviceordermonitoringlambda/util/TestDataBuilder.java

```diff
@@ -0,0 +1,185 @@
+package util;
+
+import com.amazon.vas.serviceordermonitoringlambda.model.GetJobMetricsInput;
+import com.amazon.vas.serviceordermonitoringlambda.model.GetJobMetricsInputBO;
+import com.amazon.vas.serviceordermonitoringlambda.model.GetJobMetricsOutput;
+import com.amazon.vas.serviceordermonitoringlambda.model.GetJobMetricsOutputBO;
+import com.amazon.vas.serviceordermonitoringlambda.model.JobAggregatedMetricsBO;
+import com.amazon.vas.serviceordermonitoringlambda.model.JobDetailsBO;
+import com.google.gson.Gson;
+import org.json.simple.JSONObject;
+
+import java.util.*;
+
+import static util.TestConstants.*;
+
+public class TestDataBuilder {
+
+    private final List<String> randomCities = new ArrayList<>();
+    private final List<String> randomSlotStartTime = new ArrayList<>();
+    private final List<String> randomTechnicianId = new ArrayList<>();
+    private final List<String> randomMerchantId = new ArrayList<>();
+    private final List<Integer> randomNumberOfJobs = new ArrayList<>();
+    private final int numberOfTestAggregatedJobs = 80;
+    private final long baseTime = 1592800200;
+    private final long timeGapBetweenConsecutiveTimeSlots = 3600;
+    public TestDataBuilder(){
+        generateRandomCity();
+        generateRandomSlotStartTime();
+        generateRandomMerchantId();
+        generateRandomTechnicianId();
+        generateRandomNumberOfJobs();
+
+    }
+    private void generateRandomNumberOfJobs(){
+        Random random = new Random();
+        for(int i=0;i<numberOfTestAggregatedJobs;i++)
+            randomNumberOfJobs.add(1+random.nextInt(10));
+    }
+    private void generateRandomCity(){
+        for(int i=0;i<numberOfTestAggregatedJobs;i++)
+            randomCities.add("City-"+i%10);
+    }
+    private void generateRandomMerchantId(){
+        Random random = new Random();
+        for(int i=0;i<numberOfTestAggregatedJobs;i++)
+            randomMerchantId.add("MerchantId-"+i%10);
+    }
+    private void generateRandomTechnicianId(){
+        Random random = new Random();
+        for(int i=0;i<numberOfTestAggregatedJobs;i++)
+            randomTechnicianId.add("TechnicianId-"+i%10);
```

Comment:
lots of amount of code in this class is to generate the dummy data for jobDetails. It should be straightforward but here you are writing lots of redundant code, You can save this by just creating by below approach

List<jobDetails> jobDetails = new ArrayList<>();
for( int i = 0 ; i< X ; i++ ) {
     jobDetails.add(jobDetails.city(...).merchantId(...).build());
}